### PR TITLE
{bio}[foss/2020a] Mash v2.2

### PR DESCRIPTION
**Pull request by robqiao**
(created using `eb --new-pr`)

edit: requires ~#14244~ (CapnProto)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
close/re-open to re-trigger test suite with latest `develop` now #14244 is merged

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/Mash/Mash-2.2-foss-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+easyblock = 'ConfigureMake'
+
+name = 'Mash'
+version = '2.2'
+
+homepage = 'http://mash.readthedocs.org'
+description = "Fast genome and metagenome distance estimation using MinHash"
+
+toolchain = {'name': 'foss', 'version': '2020a'}
```
[the remark this answers is not in the document]
Comment (robqiao) [reply easybuild/easyconfigs/m/Mash/Mash-2.2-foss-2020a.eb:14]: LGTM, updated. Thx @boegel

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/55f5d19d3e27d6311cd5e2f4ecc28e72 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @robqiao!

**Pull request merged**